emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/key/DynamoDBBooleanKey.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.hadoop.dynamodb.key;
```

Comment:
Missing the copyright header even though I'm not sure whether it is still needed. Someone else can confirm this.